pkg/osquery/table/launcher_info.go

```diff
@@ -36,15 +38,16 @@ func generateLauncherInfoTable(db *bbolt.DB) table.GenerateFunc {
 
 		results := []map[string]string{
 			{
-				"branch":     version.Version().Branch,
-				"build_date": version.Version().BuildDate,
-				"build_user": version.Version().BuildUser,
-				"go_version": runtime.Version(),
-				"goarch":     runtime.GOARCH,
-				"goos":       runtime.GOOS,
-				"revision":   version.Version().Revision,
-				"version":    version.Version().Version,
-				"identifier": identifier,
+				"branch":      version.Version().Branch,
+				"build_date":  version.Version().BuildDate,
+				"build_user":  version.Version().BuildUser,
+				"go_version":  runtime.Version(),
+				"goarch":      runtime.GOARCH,
+				"goos":        runtime.GOOS,
+				"revision":    version.Version().Revision,
+				"version":     version.Version().Version,
+				"identifier":  identifier,
+				"instance_id": osquery_instance_history.CurrentInstance().InstanceId,
```

Comment:
```suggestion
				"osquery_instance_id": osquery_instance_history.CurrentInstance().InstanceId,
```

I don't love how long my suggestion is, but I kinda want `osquery` in there, since it's kinda weird that we're returning osquery data on a launcher specific table. (But that's also the point)

Reply:
that makes sense, I will make the update, but I also wonder ... should we be putting the same data in two different tables ... could always just join on the most recent history table row?